x-pack/test/screenshot_creation/apps/response_ops_docs/stack_alerting/list_view.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const commonScreenshots = getService('commonScreenshots');
+  const screenshotDirectories = ['response_ops_docs', 'stack_alerting'];
+  const pageObjects = getPageObjects(['common', 'header']);
+  const rules = getService('rules');
+  const supertest = getService('supertest');
+
+  describe.only('list view', function () {
+    let serverLogConnectorId: string;
+
+    before(async () => {
+      const connectorName = `server-log-connector`;
+      serverLogConnectorId = await createServerLogConnector(connectorName);
+    });
+
+    after(async () => {
+      await deleteConnector(serverLogConnectorId);
+    });
+
+    it('connectors list screenshot', async () => {
+      await pageObjects.common.navigateToUrl(
```

Comment:
Ideally, we would add a path mapping for the connectors app in the functional test config, like this one for cases: https://github.com/elastic/kibana/blob/main/x-pack/test/functional/config.base.js#L81-L83
That way we could call it like this
```
await pageObjects.common.navigateToApp('connectors');
```

Reply:
Thanks for the tip!  Fixed in 09507663608595d70eb0c56d4366bd7cb05fe26c